cmd/terramate/cli/cli_runner_test.go

```diff
@@ -58,6 +59,14 @@ func newCLI(t *testing.T, chdir string) tscli {
 	}
 }
 
+func newCLIWithLogLevel(t *testing.T, chdir string, loglevel string) tscli {
```

Comment:
hmm instead of a new constructor we could also just create a cli + change the loglevel, like:

```go
ts := newCLI(etc)
ts.loglevel = "info"
```

The constructor is good to guarantee this is immutable (but doesn't scale well with amount of configs), just wondering what is best on this case, not sure, feel free to disregard.

Reply:
because of the problem you just said, I'd improve the API when we need more options.